docs/utils/getCSSVariablesFromStylesheet.ts

```diff
@@ -22,12 +22,20 @@ export const getCSSVariablesFromStylesheet = (varPrefix: string): Color => {
         // Access rules from the stylesheet, handle security errors when accessing cross-origin stylesheets
         return Array.from(styleSheet.cssRules);
       } catch (err) {
-        console.error(
-          'Access denied to stylesheet: ',
-          styleSheet.href,
-          '; Error: ',
-          err.message,
-        );
+        if (err instanceof Error) {
+          console.error(
+            'Access denied to stylesheet: ',
+            styleSheet.href,
+            '; Error: ',
+            err.message,
+          );
+        } else {
+          console.error(
+            'Access denied to stylesheet: ',
+            styleSheet.href,
+            '; Unknown error occurred.',
+          );
+        }
```

Comment:
So this checks if the `err` is an instance of the javascript `Error` object otherwise it logs as unknown?

Reply:
yeah this is to get around the error `'err' is of type 'unknown'`